src/ship/PlayerShipController.cpp

```diff
@@ -505,7 +505,7 @@ void PlayerShipController::StaticUpdate(const float timeStep)
 		case CONTROL_MANUAL:
 			// maintain the thrust set by the station when undocking while the timeout lasts
 			// but only in manual mode
-			PollControls(timeStep, true, mouseMotion, act);
+			PollControls(timeStep, false, mouseMotion, act);
 			Util::LimitActualSpeed(*this, timeStep, act);
 			// synchronize the rotation in 'follow orient' mode
 			if (m_followTarget && m_followMode == FOLLOW_ORI) {
```

Comment:
@azieba thanks for the fix, indeed, it can also be done this way! however, if you have follow orientation enabled, it can, on the contrary, spin the ship here, which can be disabled as follows:
```suggestion
			if (m_followTarget && m_followMode == FOLLOW_ORI && m_rotationDamping) {
```

Reply:
This might potentially turn off rotation to the followed target when dumping is off. You are right that there is something wrong here though. I think that the manual mode with fallow target on is kind of semiautomatic mode that should have the dumping turned on like with fixspeed. I will append the PR with a change for that.